examples/angular/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "lib": ["es2018", "dom"]
   },
   "angularCompilerOptions": {
+    "enableIvy": false,
```

Comment:
Why did you have to disable Ivy?

Reply:
`ng build` results in the following error without it:

```
ERROR in The target entry-point "@google-pay/button-angular" has missing dependencies:
 - @google-pay/lib/button-manager
 - @google-pay/lib/debounce
```

I suspect that this is because the `@google-pay/lib` dependency is added as a symbolic link (`file:../lib`). Advice on how to resolve would be great.

Reply:
I'll clone the repository see what can we do to not disable Ivy.

Reply:
Sounds good. The way that I've been testing is:

```sh
npm install

cd src/button-angular
npm install

cd ../..
npm run build

cd examples/angular
npm install
npm run start
```

Reply:
Sorry for the delay, had quite a few meetings today.

I tried the steps after removing `"enableIvy": false` and everything seems to be working fine.

Reply:
I've got it working after removing `"enableIvy": false` too now after doing a `git clean -dfx`. Must have been a problem with files that accumulated over time. I'm also on macOS.

One thing to note though is that I get the following warning on `ng build --prod`:

```
Building Angular Package
******************************************************************************
It is not recommended to publish Ivy libraries to NPM repositories.
Read more here: https://v9.angular.io/guide/ivy#maintaining-library-compatibility
******************************************************************************
```

How much attention should I be paying to this?

Reply:
Yes, that's a legit complain. We don't recommend publishing Ivy instructions on npm. To fix this to `tsconfig.lib.prod.json` you need to add:

```json
{
  "extends": "./tsconfig.lib.json",
  "angularCompilerOptions": {
    "enableIvy": false
  }
}
```

As you originally had. This, however, will cause issues with `@google-pay/lib` since it looks like a name which means something only inside the `src` directory from what I understand?

Reply:
> This, however, will cause issues with `@google-pay/lib` since it looks like a name which means something only inside the `src` directory from what I understand?

Yes, this is only used by the Angular component. I originally wanted a relative reference to the `lib` folder (like `../lib`) but that didn't work.

Is there another way to achieve this?

Reply:
This looks like an issue on our end when trying to export files outside the library root. I'll think of a workaround and follow up later today.

Reply:
Found a workaround which is not ideal, but will fix the issue.

Currently, even though the `rootDirs` are properly configured and the TypeScript compiler is able to resolve `@google-pay/lib`, `ng-packagr` cannot. When it finds `public-api.ts` it treats the directory the file resides in as the `rootDir`. To ensure that it treats the directory which contains both `lib` and `button-angular` as `rootDir`, you could move `public-api.ts` (maybe call it `angular-public-api`?) to `src` and update `ng-packagr.json` to:

```json
{
  "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
  "dest": "./dist",
  "lib": {
    "entryFile": "../angular-public-api.ts"
  },
  "whitelistedNonPeerDependencies": ["tslib"]
}
```

Reply:
That sounds like a good work-around. I'll give it a try.

Reply:
That worked a treat. Thanks for the suggestion.